code/modules/mining/equipment/explorer_gear.dm

```diff
@@ -130,6 +130,28 @@
 	hoodtype = /obj/item/clothing/head/hooded/cloakhood/goliath
 	body_parts_covered = CHEST|GROIN|ARMS
 
+/obj/item/clothing/suit/hooded/cloak/goliath/AltClick(mob/user)
+	. = ..()
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		if((C.get_item_by_slot(ITEM_SLOT_NECK) == src))
+			to_chat(user, span_warning("You can't adjust [src] while wearing it!"))
+			return
+		if(C.get_item_by_slot(ITEM_SLOT_OCLOTHING == src))
```

Comment:
An OR could work for this, but if you don't want to, can you use an 'else if' here? No point checking both of these since you wont be wearing it in your neck and on your exosuit slot at once

Reply:
Alright I have no idea how to fit in an 'OR' I could not find any function from that, but || seemed to do what I imagine OR would.

Reply:
|| is OR, I just find it easier to say in conversation

Reply:
Oh.